### PR TITLE
4.6.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.6.0-release](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.6.0-release)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.6.0 (2020-09-16)
+
+- Added the `theme` property to the `SkyDocsCodeExampleComponent` to allow loading Stackblitz demos in modern theme. [#130](https://github.com/blackbaud/skyux-docs-tools/pull/130)
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:3]: You can delete "allow" and change "loading" to "load."

Comment (Blackbaud-AlexKingman) [reply CHANGELOG.md]: Done!

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.6.0-release](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-4.6.0-release)

**Review by Blackbaud-DeniseCisneros — APPROVED** [review]

**Pull request merged**